backend/README.md

```diff
@@ -22,7 +22,7 @@ Once `PostgreSQL` is installed, you can create the required database and the `dj
 
 ```bash
 createdb counterfactualcovid
-psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';" # where you should replace value with the desired password
+psql counterfactualcovid -c "CREATE USER django WITH PASSWORD 'value';"  # where you should replace value with the desired password, the one for this project can be found in line 90 of /backend/settings/common.py). 
```

Comment:
You had a better option for this in a previous revision: `createuser` or similar? Let's use that instead here.

Reply:
ok implemented in 7d391d5